### PR TITLE
Fix/1552 entity switcher to menu

### PR DESCRIPTION
**Pull request by designhorf**
Changes:

- Fixed these issues https://github.com/SelfKeyFoundation/Identity-Wallet/issues/1552#issuecomment-533522104

### CODE REVIEW

**Comment by andregoncalves** [comment]
@designhorf can you open a separate issue for the username?

**Comment by designhorf** [comment]
@andregoncalves sure

**Comment by designhorf** [comment]
Created separate issue for username

File: src/renderer/wallet/main/toolbar.jsx [hunk 1]
```diff
@@ -167,10 +183,10 @@ const ProfileList = withStyles(profileStyle)(
 									)}
 								</Grid>
 								<Grid item sm={8} className={classes.profileName}>
-									<Typography variant="h6">
+									<Typography variant="subtitle1" style={{ marginBottom: '5px' }}>
```
Comment (andregoncalves) [inline src/renderer/wallet/main/toolbar.jsx]: It's better to try and avoid inline styles as much as possible.
I would suggest moving this to the profileName css rules.

**Pull request merged**